lab-said/.eslint

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
You only need one eslint file. In this case it is the eslintrc file so you can remove this one.